var/spack/repos/builtin/packages/qt-base/package.py

```diff
@@ -143,12 +144,24 @@ class QtBase(QtPackage):
             depends_on("libxrender")
 
     with when("+network"):
-        depends_on("libproxy")
+        with when("platform=linux"):
+            depends_on("libproxy")
         depends_on("openssl")
```

Comment:
```suggestion
        depends_on("openssl")
        with when("platform=linux"):
            depends_on("libproxy")
```

Super small style suggestion to put openssl before libproxy.